junit-plugin/src/main/scala/org/scalajs/junit/plugin/Compat210Component.scala

```diff
@@ -0,0 +1,71 @@
+package org.scalajs.junit.plugin
+
+import scala.reflect.internal.Flags
+import scala.tools.nsc._
+
+/** Hacks to have our source code compatible with 2.10 and 2.11.
+ *  It exposes 2.11 API in a 2.10 compiler.
+ *
+ *  @author Nicolas Stucki
+ */
+trait Compat210Component {
+
+  val global: Global
+
+  import global._
+
+  def newValDef(sym: Symbol, rhs: Tree)(
+      mods: Modifiers = Modifiers(sym.flags),
+      name: TermName = sym.name.toTermName,
+      tpt: Tree = TypeTreeMemberType(sym)): ValDef = {
+    atPos(sym.pos)(ValDef(mods, name, tpt, rhs)) setSymbol sym
+  }
+
+  def newDefDef(sym: Symbol, rhs: Tree)(
+      mods: Modifiers = Modifiers(sym.flags),
+      name: TermName = sym.name.toTermName,
+      tparams: List[TypeDef] = sym.typeParams.map(sym =>
+          newTypeDef_compat(sym, typeBoundsTree(sym))()),
+      vparamss: List[List[ValDef]] = mapParamss(sym)(sym =>
+          newValDef(sym, EmptyTree)()),
+      tpt: Tree = TypeTreeMemberType(sym)): DefDef = {
+    atPos(sym.pos)(DefDef(mods, name, tparams, vparamss, tpt, rhs)).setSymbol(sym)
+  }
```

Comment:
They do not exist (or do not have access) in 2.10.2.

I can nuke it on 2.11 and it still works.